app/src/main/java/com/duckduckgo/app/browser/BrowserWebViewClient.kt

```diff
@@ -105,27 +107,39 @@ class BrowserWebViewClient @Inject constructor(
     }
 
     override fun onPageStarted(webView: WebView, url: String?, favicon: Bitmap?) {
-        currentUrl = url
+        Timber.d("onPageStarted $url")
+
```

Comment:
Very minor comment! If you get a chance, format this file as there's some extra spacing here.

Reply:
I formatted the file; one line got formatted but wasn't near where you highlighted. Would be good to confirm your formatter makes no changes now?